loom-cli/test/cli/purs/library/loom.toml

```diff
@@ -18,15 +18,33 @@
-    , ["purescript/purescript-invariant", "tags/v1.0.0", "e0c7cd14e226de7449a843782c50cd9d03577234"]
-    , ["purescript/purescript-monoid", "tags/v1.0.0", "c12cc75643cbd9c1c066a4b964570286e7e5077f"]
-    , ["purescript/purescript-either", "tags/v1.0.0", "4b399854e8279f1ee83fcd8c265b8c02856d269b"]
-    , ["purescript/purescript-bifunctors", "tags/v1.0.0", "0eb78b63ee000077ffe1dc60097f4881800d2fd2"]
-    , ["purescript/purescript-foldable-traversable", "tags/v1.0.0", "1b8b6587e364e20c7f628fd3fbc108f065fc5802"]
+      ["purescript/purescript-prelude", "tags/v3.1.1", "c47c2f944c79dbc2ac94bc75b1bf5c3bc2714b10"]
+    , ["purescript/purescript-strings", "tags/v3.3.2", "bc9969c5aad9f09cae284848dafaedf72e591cee"]
+    , ["purescript/purescript-maybe", "tags/v3.0.0", "e899c98c12e943b502eeef4cad7e472a2328299e"]
+    , ["purescript/purescript-control", "tags/v3.3.1", "cc1df65efc4f2dbf4b2f572ddd76b34c610248eb"]
+    , ["purescript/purescript-invariant", "tags/v3.0.0", "fe250c24bbcf835af9699a2c2ddec02388933ce6"]
+    , ["purescript/purescript-monoid", "tags/v3.3.0", "27592b5ec834ce75d41caec9d348f9d46c937c81"]
+    , ["purescript/purescript-either", "tags/v3.1.0", "14880131f98f7d4f58d7230ac4dab6246465541e"]
+    , ["purescript/purescript-bifunctors", "tags/v3.0.0", "3cbba5138592dbc6b0be3eba115eb3d9ce7fdd96"]
+    , ["purescript/purescript-foldable-traversable", "tags/v3.6.1", "db20bc210f733f20a884151cfd636af5f6904303"]
+    , ["purescript/purescript-newtype", "tags/v2.0.0", "2276bd44ff5b7440c455839833c69f40cc8d8616"]
+    , ["purescript/purescript-partial", "tags/v1.2.1", "d95d52291fd0e3b9fafc722e6635e682163cbd57"]
+    , ["purescript/purescript-tailrec", "tags/v3.3.0", "4378cbc5ed6cc09efb35c08d899b359e7e2d7b49"]
+    , ["purescript/purescript-gen", "tags/v1.1.1", "56351a6354b5a4021787b6e5e2e216bc023eadc4"]
+    , ["purescript/purescript-unfoldable", "tags/v3.1.0", "eb96bd95d54da9cb2b530ff94bcc54ab5eef309e"]
+    , ["purescript/purescript-tuples", "tags/v4.1.0", "9cace46d21e38243d97496a2cee9e46c0eb26f5a"]
+    , ["purescript/purescript-arrays", "tags/v4.2.2", "8d1aab654b4b15981ebd4a48066eb1797fbcd85b"]
+    , ["purescript/purescript-unfoldable", "tags/v3.1.0", "eb96bd95d54da9cb2b530ff94bcc54ab5eef309e"]
+    , ["purescript/purescript-nonempty", "tags/v4.1.1", "4978087f102f31154ca2afaaa25475f770a7d25e"]
+    , ["purescript/purescript-distributive", "tags/v3.0.0", "c887ffa3367ca38ca966ee3b4d44dab984c16bc5"]
+    , ["purescript/purescript-type-equality", "tags/v2.1.0", "51053827f5e8df45ac5e22d393cb04999ef433d0"]
+    , ["purescript/purescript-typelevel-prelude", "tags/v2.6.0", "6e12a8e4824bdd18e2ba33a4c082adbe03244228"]
+    , ["purescript/purescript-identity", "tags/v3.1.0", "a6c43400d870a6ac6b0d27592c08f38e37350ba5"]
+    , ["purescript/purescript-st", "tags/v3.0.0", "f94763242569154264f61d343739f73d3c5045c6"]
+    , ["purescript/purescript-eff", "tags/v3.1.0", "34682c32e0d4749bc8de9c0f92abe345fe7ca975"]
```

Comment:
Just curious, why all these extra libraries just for the cli test? Are these the base standard for PS 0.11?

Reply:
These kept coming up in build failures as being required by the test. It seems that the "core" libraries have a wider net of dependencies, AFAICT, yeah.